src/app/recurring-deposits/recurring-deposits-account-view/recurring-deposits-account-view.component.html

```diff
@@ -0,0 +1,202 @@
+        <mat-card-title>
+          <h3>
+            <i class="fa fa-stop" [ngClass]="recurringDepositsAccountData.status.code | statusLookup"
+              [matTooltip]="recurringDepositsAccountData.status.value"></i>
+            Account Name : {{recurringDepositsAccountData.depositProductName}}
+          </h3>
+        </mat-card-title>
+        <mat-card-subtitle>
+          <p>
+            Account #: {{recurringDepositsAccountData.accountNo}} | Client Name:
+            {{recurringDepositsAccountData.clientName}}<br />
+            <span
+              *ngIf="!recurringDepositsAccountData.status.rejected && !recurringDepositsAccountData.status.submittedAndPendingApproval">
+              Current Balance:
+              {{recurringDepositsAccountData.currency.displaySymbol}}&nbsp;{{recurringDepositsAccountData.summary.accountBalance}}<br />
+              Available Balance:
+              {{recurringDepositsAccountData.currency.displaySymbol}}&nbsp;{{recurringDepositsAccountData.summary.availableBalance}}<br />
+            </span>
+          </p>
+        </mat-card-subtitle>
+      </div>
+
+    </mat-card-title-group>
+
+    <mat-card-actions class="account-actions">
+
+      <ng-container *ngFor="let button of buttonConfig.singleButtons">
+        <button mat-raised-button >
+          <i class="{{button.icon}}"></i> {{button.name}}</button>
+      </ng-container>
+  
+      <ng-container *ngIf="buttonConfig.options.length > 0">
+        <button mat-raised-button [matMenuTriggerFor]="More">More</button>
+        <mat-menu #More="matMenu">
+        <span *ngFor="let option of buttonConfig.options">
+          <button mat-menu-item >{{option.name}}</button>
+        </span>
+        </mat-menu>
+      </ng-container>
+
+    </mat-card-actions>
+
+  </mat-card-header>
+
+  <mat-card-content class="content">
+
+    <div class="recurringDeposits-account-tables" fxLayout="row" fxLayoutGap="2%">
+
+      <div fxFlex="49%"
+        *ngIf="!recurringDepositsAccountData.status.rejected && !recurringDepositsAccountData.status.submittedAndPendingApproval">
+        <h4 class="table-headers">Savings Details</h4>
+        <table>
+          <tbody>
+            <tr>
+              <td>Field Officer</td>
+              <td>{{recurringDepositsAccountData.fieldOfficerName}}<span
+                  *ngIf="!recurringDepositsAccountData.fieldOfficerName">Unassigned</span></td>
+            </tr>
+            <tr>
+              <td>Activation date</td>
+              <td>{{recurringDepositsAccountData.timeline.activatedOnDate | date}}</td>
+            </tr>
+            <tr>
+              <td>Maturity Date</td>
+              <td><span>{{recurringDepositsAccountData.maturityDate | date}}</span></td>
+            </tr>
+            <tr
+              *ngIf="recurringDepositsAccountData.timeline.closedOnDate || recurringDepositsAccountData.summary.totalWitddrawals">
+              <td *ngIf="recurringDepositsAccountData.timeline.closedOnDate">
```

Comment:
single ngIf on tr, also this logic in community app seems flawed to me

Reply:
Yaa.. but there could be some other params as well that could be added later. So unless we don't know the business logic better to go with the community app.